src/App.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from "react";
 import AppRoutes from "./routes";
-import "./assets/css/App.scss";
+import "./assets/authAssets/form.css";
 import { Provider } from "react-redux";
 import configureStore from "./store/";
 class App extends Component {
```

Comment:
Add a line between the imports and the class component

Reply:
noted